plugin/loader/loader.go

```diff
@@ -83,16 +86,32 @@ type PluginLoader struct {
 	state   loaderState
 	plugins map[string]plugin.Plugin
 	started []plugin.Plugin
+	config  config.Plugins
+	repo    string
 }
 
 // NewPluginLoader creates new plugin loader
-func NewPluginLoader() (*PluginLoader, error) {
-	loader := &PluginLoader{plugins: make(map[string]plugin.Plugin, len(preloadPlugins))}
+func NewPluginLoader(repo string) (*PluginLoader, error) {
```

Comment:
The repo a Plugin receives from the Environment should probably be normalized into absolute paths.
Not sure though, there may be a case for accepting relatives.

Abs patch 1/2
```suggestion
func NewPluginLoader(repo string) (*PluginLoader, error) {
	var err error
	if repo, err = filepath.Abs(repo); err != nil {
		return nil, err
	}
```

Reply:
Given that we don't do this anywhere else, I'd like to leave this as it is for consistency (that way the repo path is the same everywhere). I believe it'll _usually_ be an absolute path anyways.

If we do run into issues, we should be consistent and make this path absolute at a higher layer.